libs/application/templates/parental-leave/src/hooks/useStatefulAnswers.ts

```diff
@@ -0,0 +1,16 @@
+import { Application } from '@island.is/application/core'
+import { Dispatch, SetStateAction, useState } from 'react'
+
+import {
+  ApplicationAnswers,
+  useApplicationAnswers,
+} from './useApplicationAnswers'
+
+export const useStatefulAnswers = (
+  application: Application,
+): [ApplicationAnswers, Dispatch<SetStateAction<ApplicationAnswers>>] => {
+  const answers = useApplicationAnswers(application)
+  const [stateful, setStateful] = useState(answers)
+
+  return [stateful, setStateful]
+}
```

Comment:
Not used anywhere?

Reply:
Yes, it's used in the big file diff that's being minimized :D 
![Screenshot 2021-05-31 at 10 15 06](https://user-images.githubusercontent.com/937328/120178542-23687d80-c1f9-11eb-8921-d2a08f0a8c23.png)

Reply:
🙈